packages/core-ethereum/src/index.ts

```diff
@@ -392,24 +393,43 @@ export default class HoprCoreEthereum extends EventEmitter {
     } catch (err) {
       log(`redemption of tickets from ${channel.source.toString()} failed`, err)
     } finally {
-      delete this.ticketRedemtionInChannelOperations[channelId]
+      delete this.ticketRedemtionInChannelOperations[channelId.toHex()]
     }
   }
 
   // Private as out of order redemption will break things - redeem all at once.
```

Comment:
This comment needs to be updated. The function does not need to be private anymore.

Reply:
Ok. Will do